web-ui/src/main/java/com/tutrit/webclient/controller/CustomerController.java

```diff
@@ -14,20 +14,17 @@
 
 @Controller
 public class CustomerController {
-    @Autowired(required = false)
+
```

Comment:
changes to web-ui module are out of scope of current issue

Reply:
I agree with you. I reverted this changes